app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt

```diff
@@ -302,6 +319,132 @@ fun SearchBarImplementation(
   }
 }
 
+@Composable
+fun DisplayItineraryFromCategory(
+    itineraries: List<Itinerary>,
+    navigation: Navigation,
+    homeViewModel: HomeViewModel,
+    category: HomeCategory,
+    test: Boolean = false
+) {
+  LazyColumn(
+      modifier =
+          Modifier.fillMaxSize()
+              .padding(
+                  PaddingValues(
+                      0.dp, 0.dp, 0.dp, 70.dp)) // this ensures having a padding at the bottom
```

Comment:
Can this padding cause issues on the small phone ? It seems that it could

Reply:
I checked and it doesn't apparently, but I can put some more modular values

Reply:
If it's fine then no changes are needed. It was just to be sure !